helmfile/values/grafana/grafana-dashboards.yaml.gotmpl

```diff
@@ -9,6 +9,12 @@
 dashboard:
   ck8sVersion: {{ .Values.global.ck8sVersion }}
   extraTextGrafana: {{ .Values.welcomingDashboard.extraTextGrafana }}
+{{ if .Values.welcomingDashboard.extraVersions }}
+  extraVersions:
+{{- toYaml .Values.welcomingDashboard.extraVersions | nindent 2 }}
+{{ else }}
+  extraVersions: []
```

Comment:
```suggestion

```
Did this do anything for you? For me when i tested it with or without this part it made no difference from what i could see.

Reply:
Extra safety in case it is somehow missing from the input, but I may have gotten lost in all the layers of templating. Does it do nothing even if the corresponding line in `values.yaml` and `sc-config.yaml` is disabled?